target/target_test.go

```diff
@@ -0,0 +1,63 @@
+package mg
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+)
+
+func TestPath(t *testing.T) {
+	t.Parallel()
+	dirs := []string{"testdata/src_dir", "testdata/target_dir"}
+	for _, d := range dirs {
+		os.MkdirAll(d, 0777)
+		time.Sleep(10 * time.Millisecond)
+	}
+	files := []string{
+		"testdata/src_dir/some_file",
+		"testdata/src_dir/some_",
+		"testdata/src_file",
+		"testdata/target_dir/some_file",
+		"testdata/target_file",
+	}
+	for _, v := range files {
+		err := ioutil.WriteFile(v, []byte(v), 0600)
+		if err != nil {
+			t.Fatal(err)
+		}
+		time.Sleep(time.Second)
```

Comment:
can you make this time.Millisecond*10 or something? it doesn't have to be a second to get different timestamps.